test/index_test.js

```diff
@@ -7,6 +7,10 @@ import UkModulusChecking from './../src';
 
 const accounts = {
   invalid: [
+    { accountNumber: '12345', sortCode: '123456' },
```

Comment:
After discussing offline with @franciscocardoso, we'd prefer to have these special cases as actual tests.